apps/SageAccountWeb/src/components/useSourceApp.tsx

```diff
@@ -0,0 +1,25 @@
+import { sourceAppConfigTableID } from '../resources'
+import { SourceAppConfig } from './SourceAppConfigs'
+import { useSourceAppConfigs } from './useSourceAppConfigs'
+
+export const useSourceApp = (
+  targetSourceAppId?: string,
+): SourceAppConfig | undefined => {
+  const sourceAppId = targetSourceAppId ?? localStorage.getItem('sourceAppId')
+  const sourceAppConfigs = useSourceAppConfigs()
+
+  // PORTALS-2746: Find target source app.  Fallback to Sage Bionetworks source app if target not found.
```

Comment:
Here's the actual change.  Get the SAGE source app as the default.  If sourceApp is not found, send a message to console.error (and set the default app in local storage for future calls)